testdata/simple/5a75f537726934e4b62833ab6d5dca41/tests/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Would it work for the logs and metadata for each test to reside under a test id prefix? This could enable us to filter to just a singe test's logs without reading the metadata for each test.

Reply:
Oh yeah I think I screwed this up from the document specification, sorry.

Reply:
Should be in a new folder now.